### ISSUE TITLE
"Stick to the Front Page" should use a toggle instead of a checkmark

### ISSUE THREAD

**Issue by melchoyce** [WordPress/gutenberg#10290]
**Is your feature request related to a problem? Please describe.**
We using toggles to indicate on/off. I think that the sticky option is an on/off option, and should use a toggle instead of a checkmark.

**Describe the solution you'd like**
Change the action from a checkmark to a toggle.

![image](https://user-images.githubusercontent.com/2846578/46363252-7fda0000-c640-11e8-92e6-1c11c36ab9ce.png)

Related: #2146, #3072

**Comment by chrisvanpatten** [comment]
This was actually an intentional decision @jasmussen had made in #8588.

Quoting from there:

> The Switch is born from mobile, which is increasingly the device of choice for people and long term might become the only choice. It's bigger and more tappable, and implies by its nature that it's a boolean choice. It also has qualities that the checkbox does not embody:
> 
> + Like its wall lightswitch counterpart, it’s something you flip and the effect is instant. There used to be darkness, now there’s light.
> + It has only a single purpose, and that purpose is boolean. For example you wouldn’t use switches to select multiple categories.
> + However "Allow Comments" and the three other aforementioned toggles that use switches in master, do not have an instant effect, they require you to hit "Save" before the toggle option is stored. Given the above reasoning, a checkbox is a better choice in those situations.
> 
> The toggle is still a good choice in situations where immediate visual feedback is given, like when a dropcap is applied, or when you toggle image croppping for gallery items. In those situations, the pairing of the boolean nature and the larger touch target makes the switch feel more substantial.

(Just adding the context, I have ~no~ only a small opinion in this one!)

**Comment by jasmussen** [comment]
Yes indeed!

This is, as with all things, something we can revisit. For now, as Chris so kindly alludes to, we've devised a new ruleset for when to use switches and when to use checkboxes. 

For now that ruleset suggests that since the "stick to frontpage" toggle doesn't have any immediately visible effect, and requires you to hit the "Save" button for changes to stick, we intentionally changed it from a switch to acheckbox.

I imagine as WordPress and Gutenberg is refined over the years, this ruleset can be refined and revisited — I know there are a lot of opinions on when to use this or the other.

**Comment by melchoyce** [comment]
Makes sense! Closing this issue.
